src/app.ts

```diff
@@ -13,14 +14,47 @@ interface TemperatureSummary {
   average: number
 }
 
+const readValues: TemperatureReading[] = []
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  readValues.push(...readings)
 }
 
 export function getTemperatureSummary(
```

Comment:
one of the problems with your implementation is that every time we call this method, the information is calculated again. The idea of processReadings is to process the information once and be accessible every time you need it but without recalculating everything again.